sdk/v2/system/api_client.go

```diff
@@ -0,0 +1,42 @@
+package system
+
+import (
+	"context"
+	"net/http"
+
+	rm "github.com/brigadecore/brigade/sdk/v2/internal/restmachinery"
+	"github.com/brigadecore/brigade/sdk/v2/restmachinery"
+)
+
+// APIClient is the client for system checks involving the Brigade API.
+type APIClient interface {
+	// Ping pings the API Server
+	Ping(ctx context.Context) error
+}
+
+type apiClient struct {
+	*rm.BaseClient
+}
+
+// NewAPIClient returns a client to access system-related Brigade API
+// endpoints.
+func NewAPIClient(
+	apiAddress string,
+	apiToken string,
+	opts *restmachinery.APIClientOptions,
+) APIClient {
+	return &apiClient{
+		BaseClient: rm.NewBaseClient(apiAddress, apiToken, opts),
+	}
+}
+
+func (a *apiClient) Ping(ctx context.Context) error {
```

Comment:
On the server side, the ping returns version information (which I think was a great idea, btw), but we're not doing anything with it here. Perhaps we should return a `PingResponse` object that has a `Version` field?